workspaces/simon-game/src/app/util/playSound.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Plays a sound using the Web Audio API.
+ *
+ * @param context - The audio context to use for generating the sound.
+ * @param frequency - The frequency of the sound in Hertz
+ * @param duration - The duration of the sound in milliseconds.
+ * @param volume - The volume of the sound, typically between 0 and 1.
+ */
+export default function playSound(
```

Comment:
https://www.reddit.com/r/javascript/comments/ur01fr/askjs_defaultexports_vs_namedexports/

Reply:
Interesting! I didn't even know that by using `default` you can just name the exports w/e. I for now on prefer named exports 😄